src/bootstrap/dist.rs

```diff
@@ -848,6 +848,10 @@ impl Step for PlainSourceTarball {
 
     /// Creates the plain source tarball
     fn run(self, builder: &Builder<'_>) -> GeneratedTarball {
+        // NOTE: This is a strange component in a lot of ways. It uses `src` as the target, which
+        // means neither rustup nor rustup-toolchain-install-master know how to download it.
+        // It also contains symbolic links, unlike other any other dist tarball.
+        // FIXME: Who is using this? Is it useful?
```

Comment:
IIRC, this is the tarball to download if you want to build rustc from source (e.g., in a distro setting), with all dependencies pre-vendored and such according to our lockfile. This is (I believe) used by distros and others in a similar place.